internal/pkg/policy/policy_output.go

```diff
@@ -79,42 +80,53 @@ func (p *PolicyOutput) Prepare(ctx context.Context, zlog zerolog.Logger, bulker
 				return err
 			}
 
-			if ok := setMapObj(outputMap, outputAPIKey.Agent(), p.Name, "api_key"); !ok {
-				return ErrFailInjectAPIKey
-			}
+			agent.DefaultApiKey = outputAPIKey.Agent()
 
-			if isDefault {
```

Comment:
this parameter is no longer referenced here, we should be able to remove it from the signature.